### PR TITLE
Add Auth Pages and Functionality

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Code is clean and readable.
Folder/file structure is really good as well ! 
Good to merge!

File: src/pages/authPage/authUtils.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      return { ...signupState, password: signupAction.data.password };
+    case "CONFIRM_PASSWORD_CHANGE":
+      return {
+        ...signupState,
+        confirmPassword: signupAction.data.confirmPassword,
+      };
+    case "FIRST_NAME_CHANGE":
+      return { ...signupState, firstName: signupAction.data.firstName };
+    case "LAST_NAME_CHANGE":
+      return { ...signupState, lastName: signupAction.data.lastName };
```
Comment (iRohitGaur) [inline src/pages/authPage/authUtils.js:28]: suggestion: you can keep a single action.type for all of them just change the payload.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/pages/authPage/authUtils.js (hunk 1 above)
Comment (varun13169) [reply src/pages/authPage/authUtils.js:28]: Wanted to keep logic in separate JS file

**Pull request merged**